@xen-orchestra/mixins/SslCertificate.mjs

```diff
@@ -0,0 +1,148 @@
+import { createLogger } from '@xen-orchestra/log'
+
+import acme from 'acme-client'
+import { X509Certificate } from 'crypto'
+import fs from 'fs-extra'
+
+const { debug, warn } = createLogger('xo:mixins:sslCertificate')
+
+// from https://github.com/publishlab/node-acme-client/blob/master/examples/auto.js
+
+export default class SslCertificate {
+  #app
+  #cert
+  #key
+  #express
+  #refreshing = false
```

Comment:
IMHO, it sends the incorrect signal to set this to a boolean, I think it should be set to `undefined` and have a more explicit name:

```suggestion
  #updateSslCertificatePromise
```